marketplace/backends/database.py

```diff
@@ -119,3 +114,6 @@ def size(self, name):
                 return 0
             else:
                 return int(row[0])
+
+    def _sanitize(self, data):
```

Comment:
Is this web facing, if so maybe someone should check the sanitisation - or if its standard sanitisation provide a reference?

Reply:
This was a not-so-clever hack I added due to some inconsistencies porting data from postgres to mysql that I have now resolved. This cruft method has now been removed.